docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+|Scavenge     |final     | STW              | `<gc-start>`, `<gc-end>` |weak roots scanning, reported as a complex scavenge operation (`<gc-op>`) containing specific details for each of weak root groups  |
+
+### Global GC cycle
+
+The following example shows how a global GC cycle is recorded in a `gencon` policy verbose GC log. The output is broken down into sections with supporting text to explain the GC processing that is taking place.
+
+The global GC cycle runs when the *tenure* area is close to full, which typically occurs after many partial cycles. As such, the output can be found part way down the full log. For more information about the GC Initialization section, see [Verbose GC log contents and structure ](vgclog-md#verbose-gc-log-contents-and-structure). For an example log output for a `gencon` partial cycle, see [Example - `gencon`’s default partial GC cycle](./vgclogs.md/#example-gencons-default-partial-gc-cycle).
+
+ [The global GC cycle is split into 3 increments](./vgclog.md#gc-increments-and-interleaving) that interleave with partial GC cycles. Splitting the cycle operations into the following increments means that the STW pause times can be as short as possible, with the majority of the GC work being done concurrently. The interleaving can be seen in the following example, where a partial GC cycle is logged between the start and end of the global cycle.
+
+ The following elements log the GC increments and operations of the global GC cycle:
+
+ |GC Operation         | GC increment| STW or concurrent| XML element of GC increment| Details                         |
+|---------------------|-------------|-------------------------------|--------------------------------------|-----------------------|
+|n/a - initiates cycle|initial      | STW              | `<concurrent-kickoff`        |No `<gc-op>` is logged. This increment just initiates the concurrent mark increment |
+|concurrent mark      |intermediate |concurrent                     | `<gc-start>`, `<gc-end>`     |`<concurrent-trace-info>` records the progress of the concurrent mark increment|
+|final collection     |final        | STW              | `<concurrent-global-final>`  |Triggered when the card-cleaning threshold is reached. Child element is `<concurrent-trace-info reason=””>`. Operations and sub-operations include a final concurrent mark, a sweep, and an optional class unload and compact.|
+
+The global GC cycle follows a general structure in the verbose GC log as shown. The lines are indented to help illustrate the flow and attributes and some child elements are omitted for clarity. The structure includes a partial GC cycle that starts and finishes within the global cycle:
+
+```xml
+
+<concurrent-kickoff/>       (global cycle 1st increment recorded)
+
+<exclusive-start/>          (STW pause starts)
+
+<cycle-start/>              (global cycle starts)
+
+<exclusive-end/>            (STW pause ends)
+
+
+(global cycle 2nd increment running concurrently)
```

Comment:
you can prepend: _mutator threads running and_ global cycle concurrent increment....

Reply:
**like this?**

<concurrent-kickoff/>       (global cycle 1st increment recorded)

<exclusive-start/>          (STW pause starts)

<cycle-start/>              (global cycle starts)

<exclusive-end/>            (STW pause ends)

(mutator threads running)
(global cycle concurrent increment running concurrently)


<exclusive-start/>          (STW for partial GC cycle starts)